.github/workflows/build_test_deploy.yml

```diff
@@ -181,7 +181,7 @@ jobs:
         if: ${{ steps.swc-change.outputs.SWC_CHANGE == 'yup' }}
         run: |
           cargo fmt -- --check
-          cargo clippy --all -- -D warnings
+          cargo clippy --all -- -D warnings -A deprecated
```

Comment:
Disallowing deprecated features doesn't allow us to flag features deprecated in turbopack.
But we need that to have a workflow like:
1. add better method in turbopack, deprecate old one
2. release to next.js (current code won't break as it's only deprecated)
3. update next.js code to use better method
4. remove deprecated method in turbopack